Libraries/Hotcakes.Payment/Hotcakes.Payment.csproj

```diff
@@ -65,18 +65,36 @@
     <Reference Include="AuthorizeNet">
       <HintPath>..\..\packages\AuthorizeNet.1.8.2\lib\AuthorizeNet.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Bcl.AsyncInterfaces, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Microsoft.Bcl.AsyncInterfaces.1.1.0\lib\net461\Microsoft.Bcl.AsyncInterfaces.dll</HintPath>
+    </Reference>
+    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Newtonsoft.Json.9.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
```

Comment:
There are multiple references to `Newtonsoft.Json` in this update, and the versions don't match.  We need to prevent compile-time warnings and potential exceptions/debugging issues later.  Please be sure all identical references reflect the same reference version.

Reply:
Nuget is fine to use always, as long as the version of the reference we're looking for is correct (matches what the code expects in dev and prod).  This applies to any project where we need to add an external reference. 

Local references should only be used when we can't use Nuget.  This includes forked/custom DLLs and things not in Nuget.  :)